src/mame/video/namcos2_sprite.cpp

```diff
@@ -23,10 +23,11 @@
 
 DEFINE_DEVICE_TYPE(NAMCOS2_SPRITE, namcos2_sprite_device, "namcos2_sprite", "Namco System 2 Sprites (C106,C134,C135,C146)")
 
-namcos2_sprite_device::namcos2_sprite_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock) :
+namcos2_sprite_device::namcos2_sprite_device(const machine_config& mconfig, const char* tag, device_t* owner, u32 clock) :
```

Comment:
Please don’t go changing the spacing like this – there’s a reason for putting right-associative things to the right of the space.

Reply:
visual studio defaults to this format, MAME seems to be at odds with most other things here (and annoyingly VS seems to like resetting this to defaults from time to time, I'll change it.. again)